tools/libboard/board_common/board_common.c

```diff
@@ -73,7 +73,7 @@ fpga_result read_sysfs(fpga_token token, char *sysfs_path,
 		resval = res;
 		goto out_destroy;
 	}
-	printf("size=%d \n", size);
+
```

Comment:
Could you submit this one in a separate pull request?

Reply:
Removed from Merge request